codecov.yml

```diff
@@ -0,0 +1,24 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
```

Comment:
What happens if we set `yes`?

Reply:
I followed default-yaml.
https://docs.codecov.io/docs/codecov-yaml#default-yaml

This answer says that we can control methods and macro coverage.
> for gcov coverage reports uploaded to Codecov with branch coverage, we offer control on whether or not to parse or ignore branch coverage for methods (classes), loops (for/while), conditionals (if/else), and macros.
https://community.codecov.io/t/parsers-in-codecov-yaml/1442

Reply:
Thank you for sharing the information. I'm still not sure about the options because of the lack of description by codecov community, but I think both options should be `yes` for safety-critical systems.